docs/ai/how-to/app-service-db-auth.md

```diff
@@ -0,0 +1,319 @@
+---
+title: "Authenticate and Authorize App Service to a Vector Database"
+description: "Learn how to authenticate and authorize your App Service .NET application to a vector database solution using Microsoft Entra managed identities, Key Vault, or app settings"
+author: haywoodsloan
+ms.topic: how-to
+ms.date: 04/24/2024
+zone_pivot_groups: azure-development-tool-set-two
+
+#customer intent: As a .NET developer, I want authenticate and authorize my App Service to a vector database so that I can securely add memories to the AI in my .NET application.
+
+---
+
+# Authenticate and authorize App Service to a vector database
+
+This article demonstrates how to manage the connection between your App Service .NET application and a [vector database solution](../conceptual/vector-databases.md). It covers using Microsoft Entra managed identities for supported services and securely storing connection strings for others.
+
+By adding a vector database to your application, you can enable [semantic memories](/semantic-kernel/memories/) for your AI. The [Semantic Kernel SDK](/semantic-kernel/overview) for .NET enables you to easily implement memory storage and recall using your preferred vector database solution.
+
+## Prerequisites
+
+* An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* [.NET SDK](https://dotnet.microsoft.com/download/visual-studio-sdks)
+* [`Microsoft.SemanticKernel` NuGet package](https://www.nuget.org/packages/Microsoft.SemanticKernel)
+* [`Microsoft.SemanticKernel.Plugins.Memory` NuGet package](https://www.nuget.org/packages/Microsoft.SemanticKernel.Plugins.Memory)
+* [Create and deploy a .NET application to App Service](/azure/app-service/quickstart-dotnetcore)
+* [Create and deploy a vector database solution](/semantic-kernel/memories/vector-db)
+
+## Use Microsoft Entra managed identity for authentication
+
+If a vector database service supports Microsoft Entra authentication, you can use a managed identity with your App Service to securely access your vector database without having to manually provision or rotate any secrets. For a list of Azure services that support Microsoft Entra authentication, see [Azure services that support Microsoft Entra authentication](/entra/identity/managed-identities-azure-resources/services-id-authentication-support).
+
+### Add a managed identity to App Service
```

Comment:
Is this entire section the same as in the previous article? It might be good to pull it out into a separate include file. I'm just not sure if that works with pivots, but worth a try.

Reply:
The sections are at different heading levels unfortunately, so the content is nearly the same but not quite.